mailscanner/msre_edit.php

```diff
@@ -317,14 +317,14 @@ function Show_Form($status_msg, $short_filename, $file_contents, $CONF_ruleset_k
         $desc_value = htmlentities($desc, ENT_QUOTES);
         if (preg_match('/#DISABLED#/', $rule_part['0direction'])) {
             $rule_disabled = 1;
-            $rule_action_select_options = "<option value=\"Enable\">Enable</option>\n";
+            $rule_action_select_options = '<option value=\"Enable\">' . __('enable55') . '</option>' . "\n";
```

Comment:
when using single quote, there is no more need to escape double quote:
`$rule_action_select_options = '<option value="Enable">' . __('enable55') . '</option>' . "\n";`